habitat/core/embodied_task.py

```diff
@@ -239,6 +239,45 @@ def __repr__(self):
         )
 
 
+class SequenceSpace(Space):
```

Comment:
nit: the name here is not reflective of the purpose of this class, if possible can we make this more descriptive.